src/app/containers/MostRead/index.stories.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import MostReadContainer from './Canonical';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+
+const staticMostReadURL = (service, variant) =>
+  variant !== 'default'
```

Comment:
https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-storybook-helpers/src/withServicesKnob.jsx#L41

this logic has been added because we default to `default`